CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ Our versioning strategy is as follows:
   * Promo component ([#1897](https://github.com/Sitecore/jss/pull/1897))
   * PageContent component ([#1905](https://github.com/Sitecore/jss/pull/1905))
   * Navigation component ([#1894](https://github.com/Sitecore/jss/pull/1894))
+* `[sitecore-jss]``[sitecore-jss-angular]` Default Placeholder Content for empty fields in editMode metadata - in edit mode metadata in Pages, angular package field directives will render default or custom placeholder content if the provided field is empty; base package's GenericFieldValue model is updated to accept Date type ([#1916](https://github.com/Sitecore/jss/pull/1916))
```

Comment:
* This portion
`base package's GenericFieldValue model is updated to accept Date type`
it's an improvement, not related to the "Metadata" feature or to Angular, should be added as a separate bullet point
* New "scDateEmptyFieldEditingTemplate" custom template should be highlighted in a Changelog, it's a new feature of "Metadata" integration. Currently, it's not clear that I can use this feature and pass the custom template and how to use that

Reply:
makes sense, updated